packages/workflow/src/Interfaces.ts

```diff
@@ -2901,3 +2901,25 @@ export type IPersonalizationSurveyAnswersV4 = {
 	reportedSource?: string | null;
 	reportedSourceOther?: string | null;
 };
+
+export interface CommunityNodeAttributes {
```

Comment:
This should go to `@n8n/api-types`. 
We should not add any more new types to `n8n-workflow` unless there is no other way.